solutions/kattis-catandmice.mdx

```diff
@@ -0,0 +1,108 @@
+---
+id: kattis-catandmice
+source: Kattis
+title: Cat and Mice
+author: Kevin Sheng
+---
+
+## Explanation
+
+Binary search on the minimum possible speed of Cartesian Cat.
+
+For each speed, let $\texttt{mintime}[S][e]$ be the minimum time to go from the start to end $e$, eating each mouse in the set $S$ along the way.
+If it is impossible to do so, the value will be defined as infinity.
+When processing each state, go through all the possible previous ending points and see if it's possible to reach the defined end given the previous ending time. Calculate the minimum time for the state over all valid previous endings.
```

Comment:
I think many of the explanation lines exceed 80 characters.

Reply:
i mean don't think that matters(?)

Reply:
Yep, i think it only really matters for code, since long explanation lines look the same as if you broke them up. If you want, you can use the format code button in the editor to automatically split the lines.